INSTALL.md

```diff
@@ -19,6 +19,7 @@ conda update conda
 Setup a virtual environment in conda named `qiita` by executing the following:
 
 ```bash
+conda config --add channels conda-forge
```

Comment:
@aliu104 Annie, can you confirm that conga-forge is needed for Qiita's environment? I know that it's a requirement for some of our plugins and other projects. However I'm not sure if it's needed for Qiita itself.

Reply:
@charles-cowart Hi Charles, when I would run  `conda create -q --yes -n qiita python=3.6 pip libgfortran numpy nginx` without running  `conda config --add channels conda-forge`, it tells me that I failed with repodata from current_repodata.json, will try with next reodata source. It tells me that libgfortran is not available to download. However, after I ran the `conda config --add channels conda-forge` command, I was able to install libgfortran (see attached screenshots). This seemed to be a problem among me and 2 other students, who all had newly installed versions of Ubuntu.

Not using conda-forge:
![image](https://user-images.githubusercontent.com/22651181/179366226-7dbbcadc-62b1-4afd-b5cf-f1a16e4cb2ec.png)

Using conda-forge:
![image](https://user-images.githubusercontent.com/22651181/179366257-c5e5ec17-a1c6-4e6b-800c-ec9f2d4bc3b9.png)